build/configs/rtl8721csm/scripts/rlx8721d_img2_is.ld

```diff
@@ -106,11 +109,90 @@ SECTIONS
 
 	.ram_image2.text :
 	{
+		__ram_text_start__ = .;
 		_stext = ABSOLUTE(.);
 		*(.image2.ram.text*)
 		*(.image2.net.ram.text*)
+		/*os*/
+		*os_start.o (.text*)
+		*osdep_service.o (.text*)
+		*tizenrt_service.o (.text*)
+		/*irq(svcall)*/
+		/**irq_initialize.o (.text*)*/
+		*amebad_irq.o (.text*)
+		*up_exception.o (.text*)
+		*up_svcall.o (.text*)
+		*up_switchcontext.o (.text*)
+		*up_copyfullstate.o (.text*)
+		*up_fullcontextrestore.o (.text*)
+		*up_doirq.o (.text*)
+		*irq_dispatch.o (.text*)
+		/*scheduler(for semphr)*/
+		*sched_addreadytorun.o (.text*)
+		*sched_addblocked.o (.text*)
+		*sched_addprioritized.o (.text*)
+		*sched_removereadytorun.o (.text*)
+		*sched_removeblocked.o (.text*)
+		*sched_mergepending.o (.text*)
+		*up_blocktask.o (.text*)
+		*up_unblocktask.o (.text*)
+		/*semphr*/
+		*sem_wait.o (.text*)
+		*sem_waitirq.o (.text*)
+		*sem_trywait.o (.text*)
+		*sem_tickwait.o (.text*)
+		*sem_timedwait.o (.text*)
+		*sem_post.o (.text*)
+		/*lwip*/
+		*tcp_in.o (.text*)
+		*tcp_out.o (.text*)
+		*def.o (.text*)
+		*igmp.o (.text*)
+		*pbuf.o (.text*)
+		*inet_chksum.o (.text*)
+		*api_msg.o (.text*)
+		*netbuf.o (.text*)
+		*sys_arch.o (.text*)
+		*ethernetif_tizenrt.o (.text*)
+		*lwip_intf_tizenrt.o (.text*)
+		*timeout.o (.text*)
+		*raw.o (.text*)
+		*mem.o (.text*)
+		*memp.o (.text*)
+		*netif.o (.text*)
+		*ip.o (.text*)
+		*ip4.o (.text*)
+		*ip4_addr.o (.text*)
+		*ip4_frag.o (.text*)
+		*ip6.o (.text*)
+		*ip6_addr.o (.text*)
+		*ip6_frag.o (.text*)
+		*tcp.o (.text*)
+		*udp.o (.text*)
+		*socket.o (.text*)
+		*bsd_socket_api.o (.text*)
+		*tcpip.o (.text*)
+		*ethernet.o (.text*)
+		*etharp.o (.text*)
+		*ethip6.o (.text*)
+		*netdb.o (.text*)
+		*api_lib.o (.text*)
+		*netifapi.o (.text*)
+		/*iperf*/
+		*iperf_server_api.o (.text*)
+		*iperf_api.o (.text*)
+		*iperf_tcp.o (.text*)
+		*iperf_net.o (.text*)
+		/*fs*/
+		/**fs_read.o (.text*)
+		*fs_select.o (.text*)
+		*fs_poll.o (.text*)*/
+		/*others*/
+		*lib_vikmemcpy.o (.text*)
+		*dq_rem.o (.text*)
+		*dq_addlast.o (.text*)
```

Comment:
Previously, this code was in Flash. Now, TizenRT will be linked assuming that this code is in RAM. But, are we copying this code from flash to ram during boot up?

Reply:
Yes, we are loading code from flash to ram during bootup.